engine/src/main/java/org/hibernate/search/engine/impl/MutableSearchFactoryState.java

```diff
@@ -41,9 +42,9 @@
 	private IndexingMode indexingMode;
 	private Worker worker;
 	private BackendQueueProcessor backendQueueProcessor;
-	private Map<String, FilterDef> filterDefinitions;
+	private Map<String, FilterDef> filterDefinitions = new ConcurrentHashMap<>();
```

Comment:
I agree but I'd rather keep that out of this specific change. It has the potential to go viral and get me to refactor all of it, which I should definitely do but not in the scope of this micro fix which we need ASAP.